plugins/sqs/integration/tests/setup_suite.bash

```diff
@@ -7,4 +7,5 @@ setup_suite() {
 
 libpath_relative_to_binary="$(realpath "$(dirname "$(command -v tenzir)")")/../share/tenzir/integration/lib"
 libpath_relative_to_pwd="${BATS_TEST_DIRNAME}/../../../../lib"
-export BATS_LIB_PATH=${BATS_LIB_PATH:+${BATS_LIB_PATH}:}${libpath_relative_to_binary}:${libpath_relative_to_pwd}
+libpath_relative_if_external=${BATS_TEST_DIRNAME}/../../../../tenzir/integration/lib #needed for the externalized build in docker
+export BATS_LIB_PATH=${BATS_LIB_PATH:+${BATS_LIB_PATH}:}${libpath_relative_to_binary}:${libpath_relative_to_pwd}:${libpath_relative_if_external}
```

Comment:
I don't have a better idea for fixing this, but I think you should quickly discuss this with @tobim and/or @lava, who wrote this originally.